# Supplementary information

Supplementary information with the article 'Large-scale genomic characterisation of Bacteroides fragilis reveals multiple functionally divergent lineages and a distinct species.'

## Supplementary Data

### Supplementary Table 1: Multi locus sequence typing

-   The [standard output](Supplementary_data/Multi_locus_sequence_typing/mlst.stdout) of (MLST)[https://github.com/tseemann/mlst] using the B. fragilis scheme.
-   The [novel alleles](Supplementary_data/Multi_locus_sequence_typing/mlst_novel_alleles.fasta) identified in the MLST analysis.
-   The [lineage associated Sequencing Types](Supplementary_data/Multi_locus_sequence_typing/lineage_ST.tsv).

### Supplementary Table 2: [Lineage specific genes](Supplementary_data/Lineage_specific_genes/lineage_specific_genes.tsv)

An overview table of the lineage specific genes identified in the B. fragilis pangenome. Based on the following conditions:

-   Precision (PPV) ≥ 85%: At least 85% of genomes with the gene belong to a single lineage.
-   Sensitivity ≥ 95%: At least 95% of genomes from the designated lineage possess the gene.

### Supplementary Table 3: [Co-occuring Mobile genes](Supplementary_data/Mobile_genes/coinfinder_anno_mobile.tsv)

An overview table of the mobile genes identified in the _B. fragilis_ pangenome. Based on the following conditions:

-   Co-occuring genes were identified with [Coinfinder V.1.0.8](https://github.com/fwhelan/coinfinder)
-   Genomic proximity: at least one genome exhibits inter-gene locus standard deviation ≤ 15 units
-   Lineage distribution: at least one assembly of a lineage harbors ≥ 80% of gene group, but no more than 16 out of 17 lineages contain a assembly with 80% of the gene group.

### Supplementary Table 4: [Drug resistance](Supplementary_data/Drug_resistance/abricate_summary.tsv)

An overview table of the drug resistance genes identified in the _B. fragilis_ strains using [Abricate](https://github.com/tseemann/abricate) and [NCBI AMRFinderPlus v3.10](https://doi.org/10.1128%2FAAC.00483-19).

## Supplementary Figures

### Supplementary figure 1

![B fragilis data acquisition](Supplementary_figures/supp_fig_1/Bfragilis_data_acquisition.png)

_**Supplementary Figure 1:** Overview of data preprocessing steps. †Quality control measures are: contamination ≦ 5%, completeness ≧ 90%, 4.25Mbp ≦ genome size ≦ 5.75Mbp, # contigs ≦ 600, N’s per 100kbp ≦ 100. <sup>\*</sup>Final dataset used in downstream analysis, 498 of B. fragilis sensu stricto genomes and 68 B. fragilis A genomes._

### Supplementary figure 2

### Supplementary figure 3

### Supplementary figure 4

### Supplementary figure 5

_**Supplementary Figure 5:** (**A**)Significantly associated gene cluster of lineage J containing the genes: bshA, bshB_, dxs, tilS, ispD, rmlB, rpiB, strE, tarF and 6 hypothetical proteins. Genes were highlighted within assembly GCF*00856913.1. (**B**) Significantly associated gene cluster of lineage Q containing the genes: tuaB, gmhB, epsM and arnB. Genes are highlighted within GCA_016623685.1 using SnapGene software (www.snapgene.com).*

### Supplementary figure 6
